pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.openhab.addons.bundles</groupId>
     <artifactId>org.openhab.addons.reactor.bundles</artifactId>
-    <version>4.2.0</version>
+    <version>4.3.0</version>
```

Comment:
I branched from 785776a18da51bb5c9a90778ea71382206905f01 - so I think this is ok to bring into your backport branch